app/views/welcome/_tomatoes_timer.html.erb

```diff
@@ -65,4 +65,15 @@
 <div id="start_container" class="center-block">
   <%= link_to t('timer.start'), '#', id: 'start', class: 'btn btn-default btn-lg' %>
   <p id="start_hint" class="help-block"><%= t('timer.or_press_space') %></p>
+  <div class="form-group" style="width: 50%; margin: 0 auto;">
+    <input name="duration_time"
+           id="duration_time_input"
+           class="form-control"
+           type="range"
+           min="5"
+           max="60"
+           value="5"
+           oninput="duration_time_output.value = duration_time_input.value">
+    <output style="display: inline-block;" name="duration_time_output" id="duration_time_output">5</output>
```

Comment:
It would be more clear if we add a label, _duration_, and a unit of measurement, _minutes_.